packages/workspace/src/tasks-runner/run-command.ts

```diff
@@ -21,19 +21,56 @@ import {
 } from '../utilities/project-graph-utils';
 import { output } from '../utilities/output';
 import { getDependencyConfigs } from './utils';
-import { Hasher } from '../core/hasher/hasher';
+import * as process from 'process';
```

Comment:
I imagine this was probably an erroneous auto import, we of course want to use the `process` global rather than this package